libraries/adaptive-expressions/src/triggerTrees/quantifier.ts

```diff
@@ -27,14 +27,15 @@ export enum QuantifierType {
 export class Quantifier {
     /**
      * Initializes a new instance of the `Quantifier` class.
+     *
      * @param variable Name of variable to replace.
      * @param type Type of quantifier.
      * @param bindings Possible bindings for variable.
      */
     public constructor(readonly variable: string, readonly type: QuantifierType, readonly bindings: string[]) {}
 
     /**
-     * Returns a string that represents the quantifier.
+     * @returns a string that represents the quantifier.
```

Comment:
Start the description of the return with uppercase

Reply:
Done. Thanks!